src/main/java/com/limechain/runtime/babeapi/BabeApiConfiguration.java

```diff
@@ -0,0 +1,25 @@
+package com.limechain.runtime.babeapi;
+
+import com.limechain.chain.lightsyncstate.Authority;
+import com.limechain.chain.lightsyncstate.BabeEpoch;
+import lombok.Getter;
+import lombok.Setter;
+import org.javatuples.Pair;
+
+import java.math.BigInteger;
+import java.util.List;
+
+
+/**
+ * Current configuration of the BABE consensus protocol.
+ */
+@Getter
+@Setter
```

Comment:
```suggestion
@Data
```
[Data](https://projectlombok.org/features/Data) would be more appropriate here. Keeps the same getter/setter with no arg setup, but also adds toString and Equals for tests etc.